db/document.go

```diff
@@ -65,6 +65,7 @@ type SyncData struct {
 	Expiry          *time.Time          `json:"exp,omitempty"`           // Document expiry.  Information only - actual expiry/delete handling is done by bucket storage.  Needs to be pointer for omitempty to work (see https://github.com/golang/go/issues/4357)
 	Cas             string              `json:"cas"`                     // String representation of a cas value, populated via macro expansion
 	Crc32c          string              `json:"value_crc32c"`            // String representation of crc32c hash of doc body, populated via macro expansion
+	Crc32cUserXattr string              `json:"user_xattr_value_crc32c"` // String representation of crc32c hash of user xattr
```

Comment:
I expected omitempty here - I don't think we need to set the property when empty, correct?